returnn/engine/base.py

```diff
@@ -148,6 +148,9 @@ def get_epoch_model(cls, config):
                 config.value("load", ""),
                 load_model_epoch_filename + cls.get_file_postfix(),
             )
+            # If "load" is given and "model" is not, always load explicitly
```

Comment:
Wouldn't a check for "config.value("task", "train") != train" be clearer here? Because isn't that what is checked here in the end? Or is there another case outside of training where `load` and `model` are set?
When I look at the `if else` block below the case where `load` and `model` are set outside of training would be caught in the first condition and the fact that "model" is set or not does not change anything about the behaviour of that block, right?